MaxText/scratch_code/golden_llama3-8b_export.ipynb

```diff
@@ -0,0 +1,234 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Defaulting to user installation because normal site-packages is not writeable\n",
+      "Looking in indexes: https://download.pytorch.org/whl/cpu\n",
+      "Requirement already satisfied: torch in /home/mohitkhatwani/.local/lib/python3.10/site-packages (2.3.0)\n",
+      "Collecting torchvision\n",
```

Comment:
it is generally recommended to not put the outputs of the notebook cells into the repo